site/source/docs/porting/simd.rst

```diff
@@ -1025,144 +1025,291 @@ Emscripten supports compiling existing codebases that use ARM NEON by
+<https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics>`_.
 
-.. list-table:: NEON Intrinsic classes available
-   :widths: 20
+.. list-table:: NEON Intrinsics
+   :widths: 20 30
    :header-rows: 1
- * - ❌ vaba
- * - ❌ vabal
- * - ❌ vabd
- * - ❌ vabdl
- * - ✅ vabs
- * - ✅ vadd
- * - ❌ vaddl
- * - ❌ vaddlv
- * - ❌ vaddv
- * - ✅ vaddw 
- * - ✅ vand 
- * - ❌ vbic
- * - ✅ vbsl
- * - ✅ vcagt
- * - ✅ vceq
- * - ✅ vceqz
- * - ✅ vcge
- * - ✅ vcgez
- * - ✅ vcgt
- * - ✅ vcgtz
- * - ✅ vcle
- * - ✅ vclez
- * - ❌ vcls
- * - ✅ vclt
- * - ✅ vcltz 
- * - ❌ vcnt
- * - ❌ vclz
- * - ✅ vcombine 
- * - ✅ vcreate
- * - ✅ vdot
- * - ✅ vdot_lane
- * - ❌ vdup
- * - ✅ vdup_n
- * - ✅ veor
- * - ❌ vext
- * - ✅ vget_lane
- * - ❌ vhadd
- * - ❌ vhsub
- * - ✅ vld1
- * - ❌ vld2
- * - ❌ vld3
- * - ❌ vld4
- * - ❌ vmax
- * - ❌ vmaxv
- * - ✅ vmin
- * - ❌ vminv
- * - ✅ vmla 
- * - ❌ vmlal
- * - ❌ vmls
- * - ❌ vmlsl
- * - ❌ vmovl
- * - ✅ vmul
- * - ✅ vmull 
- * - ❌ vmull_n
- * - ❌ vmull_high
- * - ✅ vmvn
- * - ✅ vneg
- * - ❌ vorn
- * - ✅ vorr
- * - ✅ vpadal
- * - ✅ vpadd
- * - ✅ vpaddl 
- * - ❌ vpmax
- * - ✅ vpmin
- * - ❌ vpminnm
- * - ❌ vqabs
- * - ❌ vqabsb
- * - ✅ vqadd 
- * - ❌ vqaddb
- * - ✅ vqdmulh  
- * - ❌ vqneg
- * - ❌ vqnegb
- * - ❌ vqrdmulh
- * - ❌ vqrshl
- * - ❌ vqrshlb
- * - ❌ vqshl
- * - ❌ vqshlb
- * - ❌ vqsub
- * - ❌ vqsubb
- * - ✅ vqtbl1
- * - ✅ vqtbl2
- * - ✅ vqtbl3
- * - ✅ vqtbl4
- * - ✅ vqtbx1
- * - ✅ vqtbx2
- * - ✅ vqtbx3
- * - ✅ vqtbx4
- * - ✅ vrbit
- * - ✅ vreinterpret
- * - ✅ vrev16
- * - ✅ vrev32
- * - ✅ vrev64
- * - ✅ vrhadd
- * - ✅ vrshl
- * - ✅ vrshr_n
- * - ✅ vrsra_n
- * - ✅ vset_lane
- * - ✅ vshl
- * - ✅ vshl_n
- * - ✅ vshr_n
- * - ✅ vsra_n
- * - ✅ vst1
- * - ✅ vst1_lane
- * - ❌ vst2
- * - ❌ vst3
- * - ❌ vst4
- * - ✅ vsub
- * - ❌ vsubl
- * - ❌ vsubw
- * - ✅ vtbl1
- * - ✅ vtbl2
- * - ✅ vtbl3
- * - ✅ vtbl4
- * - ✅ vtbx1
- * - ✅ vtbx2
- * - ✅ vtbx3
- * - ✅ vtbx4
- * - ✅ vtrn
- * - ✅ vtrn1
- * - ✅ vtrn2
- * - ✅ vtst
- * - ❌ vuqadd
- * - ❌ vuqaddb
- * - ✅ vuzp
- * - ✅ vuzp1
- * - ✅ vuzp2
- * - ✅ vzip
- * - ✅ vzip1
- * - ✅ vzip2
+
+   * - Intrinsic name
+     - Wasm SIMD Support
+   * - vaba
+     - ⚫ Not implemented, will trigger compiler error
+   * - vabal
+     - ⚫ Not implemented, will trigger compiler error
+   * - vabd
+     - ⚫ Not implemented, will trigger compiler error
+   * - vabdl
+     - ⚫ Not implemented, will trigger compiler error
+   * - vabs
+     - native
+   * - vadd
+     - native
+   * - vaddl
+     - ⚫ Not implemented, will trigger compiler error
+   * - vaddlv
+     - ⚫ Not implemented, will trigger compiler error
+   * - vaddv
+     - ⚫ Not implemented, will trigger compiler error
+   * - vaddw 
+     - ❌ Will be emulated with slow instructions, or scalarized
+   * - vand 
+     - native
+   * - vbic
+     - ⚫ Not implemented, will trigger compiler error
+   * - vbsl
+     - native
+   * - vcagt
+     - ⚠ Does not have direct implementation, but is emulated using fast NEON instructions
```

Comment:
This caution emoji renders differently from the one in the legend for me (and the one from the legend looks better). Same with all the other caution emojis in this list.